### DISCUSSION TITLE
Specific audio source for specific rtmp

### DISCUSSION THREAD

**Discussion by mredodos** [sorayuki/obs-multi-rtmp#153] [Ideas]
Hi, is possible send specific audio source to specific rtmp?

Because Facebook and YouTube not accept music in the stream and I want put just microphone for that rtmp 

Best regards

**Comment by sorayuki** [comment]
Hi, you can use the audio mixer in OBS to streaming different audio tracks to different target. In this plugin you can specify which mixer id to encode

**Reply by mredodos** [reply]
hi, thx for the answer but i dont understand how i can make it, can you explain better pls ?
